detectron2/modeling/matcher.py

```diff
@@ -92,7 +98,7 @@ def __call__(self, match_quality_matrix):
 
         for (l, low, high) in zip(self.labels, self.thresholds[:-1], self.thresholds[1:]):
             low_high = (matched_vals >= low) & (matched_vals < high)
-            match_labels[low_high] = l
+            match_labels.masked_fill_(low_high, l)
```

Comment:
can you link to a pytorch issue about the lack of indexing?

Reply:
done